src/wallet/wallet.cpp

```diff
@@ -2546,6 +2546,7 @@ CWallet::Balance CWallet::GetBalance(const int min_depth, const bool avoid_reuse
 {
     Balance ret;
     isminefilter reuse_filter = avoid_reuse ? ISMINE_NO : ISMINE_USED;
+    const bool cj_enable = CCoinJoinClientOptions::IsEnabled();
```

Comment:
312319f4d3de1ce3d71cb90d8d213119e1cdbb79: nit:
```suggestion
    const bool cj_enabled = CCoinJoinClientOptions::IsEnabled();
```